### DISCUSSION TITLE
wayland

### DISCUSSION THREAD

**Discussion by aloispichler** [jpalardy/vim-slime#393] [General]
Thank you for slime, I am using it regularly.
Does it work on wayland?

**Comment by jpalardy** [comment]
Hi @aloispichler 

I don't see why not. It all depends which target (tmux, wezterm, …) you're using.

Let me know how that goes.

**Reply by aloispichler** [reply]
I use slime for sending code (julia, say) from vim (in the terminal) to another terminal to execute the code. This works nice with X11, but this work flow did not work in Wayland. And this is why I put this question here.

**Reply by jpalardy** [reply]
Ah… sorry. I don't use the `x11` target myself.

You can trying cc'ing the plugin's contributor ([link](https://github.com/jpalardy/vim-slime/blame/main/autoload/slime.vim#L349-L362))
